### PR TITLE
Cleanup unused js

### PR DESCRIPTION
**Pull request by nschonni**
Remove unrefereneced scripts and packages

### CODE REVIEW

File: JabbR/JabbR.csproj [hunk 1]
```diff
@@ -820,7 +819,6 @@
     <Content Include="Scripts\jquery.timeago.0.10.js" />
     <Content Include="Scripts\jQuery.tmpl.js" />
     <Content Include="Scripts\jQuery.tmpl.min.js" />
-    <Content Include="Scripts\jquery.writeCapture.js" />
```
Comment (davidfowl) [inline JabbR/JabbR.csproj:823]: You sure this isn't used?

Comment (samandmoore) [inline JabbR/JabbR.csproj:823]: I was pretty sure this was used by one of the content providers.

Comment (nschonni) [inline JabbR/JabbR.csproj:823]: I couldn't find anything referencing it. I believe they all use jquery.captureDocumentWrite.js now

Comment (samandmoore) [inline JabbR/JabbR.csproj:823]: Ah. Right. that makes sense. Well, as long as you've verified things.

Comment (samandmoore) [inline JabbR/JabbR.csproj:823]: @nschonni pretty sure you're right on that one. lgtm.

**Pull request merged**